subprojects/docs/src/docs/release/notes.md

```diff
@@ -61,6 +61,10 @@ You may now force Gradle to use rich or plain [build output](userguide/console.h
 
 The JaCoCo plugin has been upgraded to use [JaCoCo version 0.7.9](http://www.jacoco.org/jacoco/trunk/doc/changes.html) by default.
 
+### Adjustments in the `eclipse` plugin
```

Comment:
I'd call out the feature in the headline: "Eclipse plugin separates output folders"

Reply:
Fixed